src/argilla/server/schemas/v1/datasets.py

```diff
@@ -347,3 +347,12 @@ def check_user_id_is_unique(cls, values):
 
 class RecordsCreate(BaseModel):
     items: conlist(item_type=RecordCreate, min_items=RECORDS_CREATE_MIN_ITEMS, max_items=RECORDS_CREATE_MAX_ITEMS)
+
+
+class SearchRecord(BaseModel):
+    record: Record
+    query_score: float
```

Comment:
It makes sense to be more flexible on score restrictions and let the possibility of no scores at all.

```suggestion
    query_score: Optional[float]
```